posthog/warehouse/data_load/service.py

```diff
@@ -66,6 +66,17 @@ def get_sync_schedule(external_data_schema: ExternalDataSchema):
     )
 
 
+def get_sync_frequency(external_data_schema: ExternalDataSchema):
+    if external_data_schema.source.sync_frequency == ExternalDataSource.SyncFrequency.DAILY:
+        return timedelta(days=1)
+    elif external_data_schema.source.sync_frequency == ExternalDataSource.SyncFrequency.WEEKLY:
+        return timedelta(weeks=1)
+    elif external_data_schema.source.sync_frequency == ExternalDataSource.SyncFrequency.MONTHLY:
+        return timedelta(weeks=4)
```

Comment:
4 weeks !== monthly - can we do `months=1`, or do `days=365/12` ?

Reply:
how about 30 days? months isn't a concept for timedelta